Makefile

```diff
@@ -178,6 +178,7 @@ $(build_sysconfdir)/julia/juliarc.jl: $(JULIAHOME)/contrib/windows/juliarc.jl
 endif
 
 $(build_private_libdir)/%.$(SHLIB_EXT): $(build_private_libdir)/%.o
+	objcopy --globalize-symbol __LLVM_StackMaps $<
```

Comment:
osx binutils doesn't have objcopy, and for cross-compile it would need the cross prefix